scrapy_autoextract/providers.py

```diff
@@ -55,11 +55,17 @@ def _stop_if_account_disabled(exception: Exception, crawler: Crawler):
 
 
 def get_concurrent_requests_per_domain(settings: Settings):
-    """Return the configured concurrent request per domain from settings"""
-    concurrency = settings.getint("CONCURRENT_REQUESTS_PER_DOMAIN", 8)
+    """Return the configured AutoExtract concurrent request per domain from settings"""
+    limit_name = "AUTOEXTRACT_CONCURRENT_REQUESTS_PER_DOMAIN"
+    concurrency = settings.getint(limit_name, -1)
+    # If no AutoExtract-specific limit is provided - use the default one
+    if concurrency is -1:
```

Comment:
```suggestion
    if concurrency == -1:
```

It is a CPython implementation detail that some int numbers are always the same object; better not to rely on it.